tools/mesmer/mesmer.xml

```diff
@@ -1,28 +1,26 @@
-<tool id="mesmer" name="Mesmer" version="@VERSION@.0" profile="17.09">
+<tool id="mesmer" name="Mesmer" version="@VERSION@.0" profile="21.05">
```

Comment:
Recommend 19.01 to be compatible with OHSU galaxy infrastructure.

Reply:
Ah, I see. Please note that this version is not supported anymore afaik. If you like I can change it.

Reply:
Understood, yes please change.

Reply:
done